README.md

```diff
@@ -1 +1,84 @@
 # hof-lookup-replacer
+
+A Node.js script for replacing db lookup tables as a scheduled job configurable to different HOF form services.
+
+The script retrieves a CSV formatted file from a given location, validates against given rules and then replaces a lookup table using the CSV rows. On success or failure the script will send appropriate notification messages.
+
+This service uses either [pg-promise](https://vitaly-t.github.io/pg-promise/index.html) or [knex.js](https://knexjs.org/) to handle db connection and query building. You can configure the client and model used using environment variables.
+
+## Script steps
+
+1. Scheduled job runs with service specific environment.
+2. Connect to database and retrieve the latest uploaded data's S3/file-vault URL from an SQL database.
+3. Authenticate with the service's file-vault.
+4. Use the data URL to bring the CSV file data into the application.
+5. Validate the CSV rows against rules given for the service. Log, note and remove erroring rows.
+6. Replace the SQL db's lookup table with the validated rows from CSV.
+7. Prepare a and send a notification email of pass or fail including erroring rows noted in validation.
```

Comment:
```suggestion
7. Prepare and send a notification email. Confirm a success or failure of the job. If it fails include the rows that have failed.
```
Would the above be a bit more clearer?

Reply:
Not exactly in that we won't automatically fail the job for a few invalid rows in the whole batch, have updated this line anyway to try to be more clear

Reply:
I would probably just reword the sentence to be a bit more clear.  I prefer success or fail.  Pass can mean pass as in football but if you like this
```Prepare and send a notification email. This will provide a pass or fail including erroring rows.```